lib/constants/json-schema.js

```diff
@@ -0,0 +1,268 @@
+/**
+ * The JSON Schema properties.
+ */
+const JSON_SCHEMA = [
+  {
+    name: 'bsonType',
+    value: 'bsonType',
+    label: 'bsonType',
+    score: 1,
+    meta: 'json-schema',
+    version: '0.0.0',
```

Comment:
```suggestion
    version: '3.6.0',
```

Also for the rest of the operators below.

Reply:
Done!